Source/FlowEditor/Private/Graph/FlowGraphPinFactory.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "Graph/FlowGraphPinFactory.h"
+#include "Graph/FlowGraphSchema.h"
+#include "Graph/FlowGraphSettings.h"
+#include "Graph/Nodes/FlowGraphNode.h"
+#include "Graph/Widgets/SFlowGraphNode.h"
+#include "Nodes/FlowNode.h"
+#include "Nodes/FlowPin.h"
+
+#include "NodeFactory.h"
+#include "SGraphPin.h"
+
+#include UE_INLINE_GENERATED_CPP_BY_NAME(FlowGraphPinFactory)
+
+//////////////////////////////////////////////////////////////////////////
+// FFlowGraphPinFactory
+
+TSharedPtr<SGraphPin> FFlowGraphPinFactory::CreatePin(UEdGraphPin* InPin) const
```

Comment:
I moved the pin generation out into a PinFactory to match how UE's K2 schema was doing it.  This previously was done in the widget code (SFlowGraphNode.cpp)